src/osgPlugins/OpenFlight/VertexPaletteManager.h

```diff
@@ -53,7 +54,7 @@ class VertexPaletteManager : public osg::Referenced
 
     unsigned int byteOffset( unsigned int idx ) const;
 
-    void write( DataOutputStream& dos ) const;
```

Comment:
This and the removal of the mutable on verticesStr are the only change we should take. The rest of the changes we should take the 3.6 version instead.

Reply:
Roger.  Against previous comment in the .cpp file, I'll remove mutable on verticesStr as well as const on write().   Everything else will be 3.6 changes.

Reply:
Will revert changes and:
1) Remove const from write().
2) Remove mutable from verticesStr().